mkdocs.yml

```diff
@@ -4,12 +4,14 @@ repo_url: https://github.com/srobo/runbook
 strict: true
 edit_uri: blob/master/docs/
 theme:
-  logo:
-    icon: library_books
-  favicon: favicon.ico
-  name: material
-  feature:
-    tabs: true
+  name: 'material'
+  icon:
+    logo: fontawesome/solid/book
+  favicon: 'favicon.ico'
+  language: 'en'
+  features:
+    - navigation.tabs
+    - navigation.instant
```

Comment:
Suggestion: We should add the [`navigation.index`](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#section-index-pages) feature. It means the dropdowns are clickable and have navigation, which will be incredibly helpful for some of the content we write.

Reply:
Agree, though perhaps not as part of this upgrade PR?